src/main/java/org/openhab/automation/jrule/rules/value/JRulePercentValue.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.automation.jrule.rules.value;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+
+/**
+ * The {@link JRulePercentValue}
+ *
+ * @author Joseph (Seaside) Hagberg - Initial contribution
+ */
+public class JRulePercentValue implements JRuleValue {
```

Comment:
Should this not extend JRuleDecimalValue to align with openhab internals?

Reply:
done